### DISCUSSION TITLE
Add Faster Linter Plugin using Deno linter

### DISCUSSION THREAD

**Discussion by forest1102** [web-infra-dev/rspack#8496] [Ideas]
Currently, I am using ForkTsCheckerWebpackPlugin for linting when build-dev.  
However, when using Deno, ForkTsCheckerWebpackPlugin does not work due to Deno and Node compatibility.  
And `deno_lint` is developed by Rust and published in crate. 
So, DenoLinterRspackPlugin will become faster alternative.

**Comment by hardfist** [comment]
feel free to make one using js plugin

**Reply by forest1102** [reply]
Thanks
Can I make the plugin in this repository? Or should I create own repo?

**Reply by hardfist** [reply]
you can create your own
